More cleanups of main man page kpod-1.md and README.md

rhatdan opened the pull request:
Also added new commands to transfer.md

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

TomSweeneyRedHat commented on README.md:
```diff
@@ -39,21 +39,24 @@ It is currently in active development in the Kubernetes community through the [d
+| [kpod-export(1)](/docs/kpod-export.1.md)             | Export container's filesystem contents as a tar archive                                              |
+| [kpod-history(1)](/docs/kpod-history.1.md)           | Shows the history of an image                                                                        |
+| [kpod-images(1)](/docs/kpod-images.1.md)             | List images in local storage                                                                         |
+| [kpod-info(1)](/docs/kpod-info.1.md)                 | Display System Information                                                                           |
```
system information?  (lower)

TomSweeneyRedHat commented on README.md:
```diff
@@ -39,21 +39,24 @@ It is currently in active development in the Kubernetes community through the [d
+| [kpod-save(1)](/docs/kpod-save.1.md)                 | Saves an image to an archive                                                                         |
+| [kpod-tag(1)](/docs/kpod-tag.1.md)                   | Add an additional name to a local image                                                              |
+| [kpod-umount(1)](/docs/kpod-umount.1.md)             | Unmount a working container's root filesystem                                                        |
+| [kpod-version(1)](/docs/kpod-version.1.md)           | Display the Kpod Version Information                                                                 |
```
version information (lower)

TomSweeneyRedHat commented on docs/kpod.1.md:
```diff
@@ -46,17 +46,50 @@ has the capability to debug pods/images created by crio.
+Unmount a working container's root file system
+
+### version
+Display the KPOD Version Information
```
version information (lower)

rhatdan commented:
@runcom As soon as these are reviewed, please merge, since no reason to run tests.

TomSweeneyRedHat commented on transfer.md:
```diff
@@ -39,11 +39,16 @@ There are other equivalents for these tools
-| `docker build` | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
-| `docker push` | [`buildah push`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-push.md) |
+| `docker build`  | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
+| `docker cp`     | [`kpod mount`](./docs/kpod-cp.md)        |
```
missing a '.1' in the link.  (.docs/kpod-cp.1.md)

rhatdan replied:
fixed

nalind commented on README.md:
```diff
@@ -39,21 +39,24 @@ It is currently in active development in the Kubernetes community through the [d
-| [kpod-tag(1)](/docs/kpod-tag.1.md)         | Add an additional name to a local image |
-| [kpod-umount(1)](/docs/kpod-umount.1.md)   | Unmount a working container's root filesystem |
-| [kpod-version(1)](/docs/kpod-version.1.md) | Display the Kpod Version Information |
+| [crio(8)](/docs/crio.8.md)                           | Enable OCI Kubernetes Container Runtime daemon                                                       |
```
This isn't a new line, but `Enable` doesn't seem to belong there.

nalind commented on transfer.md:
```diff
@@ -39,11 +39,16 @@ There are other equivalents for these tools
-| `docker push` | [`buildah push`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-push.md) |
+| `docker build`  | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
+| `docker cp`     | [`kpod mount`](./docs/kpod-cp.1.md)       |
+| `docker images` | [`kpod images`](./docs/kpod-images.1.md)  |
```
This isn't in the right place if we're sorting alphabetically.

nalind commented:
Nits, otherwise LGTM.

rhatdan commented:
@nalind @TomSweeneyRedHat I think I have fixed everything, also added a link to the transfer page from the README.md.
I think we should merge and then continue cleanups in future pull requests.

rhatdan commented:
@mrunalp Please merge, since this is documentation change only.  No reason to run tests.
